core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/JDBCSourceAsColumnarStore.scala

```diff
@@ -179,6 +290,10 @@ class JDBCSourceAsColumnarStore(override val connProperties: ConnectionPropertie
           stmt.close()
         } catch {
           // TODO: test this code
+          //TODO:Suranjan This code is no needed now.
+          // Need to rollback.
```

Comment:
Rollback will automatically happen through is success is "false" in ColumninsertExec.
This was old code, to try to clean already inserted rows/entries from table in case of some failure. Don't think that would have worked anyway. 
Will be removing this code of deleting entries in case of failure completely. Wanted @sumwale opinion so left that.